source/write/replace.txt

```diff
@@ -0,0 +1,282 @@
+
+Replace One Document Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``replace_one()`` method to replace 
+a document that has a ``name`` field value of ``Nobu`` with a new document 
+that has a ``name`` field value of ``La Bernadin``:
+
+.. literalinclude:: /includes/write/replace.cpp
+   :start-after: start-replace-one
+   :end-before: end-replace-one
+   :language: cpp
+   :dedent:
+
+To check if you successfully replaced the document, you can use the :ref:`find_one() <cpp-retrieve-find-one>` method to print out the new document:
+
+.. io-code-block::
+
+   .. input:: /includes/write/replace.cpp
+      :start-after: start-replace-one-io
+      :end-before: end-replace-one-io
+      :language: cpp
+      :dedent:
+   
+   .. output:: 
+      :language: cli
+      :visible: false
+      
+      New document: { "_id" : { "$oid" : "5eb3d668b31de5d588f42e34" }, "name" : "La Bernadin" }  
+
+Options 
+~~~~~~~
+
+You can modify the behavior of the ``replace_one()`` method by
+passing an instance of the ``mongocxx::options::replace`` class as an optional
+argument. The following table describes the fields you can set in a
+``mongocxx::options::update`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Function 
+     - Description
+
+   * - ``bypass_document_validation``
+     - | Specifies whether the replace operation bypasses document validation. When set to ``true``, this lets you replace a document with a new document that doesn't meet the schema validation requirements. 
+         For more information, see :manual:`Schema Validation </core/schema-validation/#schema-validation>` in the {+mdb-server+} manual.
+       | Defaults to ``false``.
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``comment``
+     - | Specifies a comment of any valid BSON type to attach to the operation. 
+         Once set, this comment appears alongside records of this command in the following locations:
+
+       - :ref:`mongod log messages <log-messages-ref>`, in the
+         ``attr.command.cursor.comment`` field
+       - :ref:`Database profiler output <profiler>`, in the :data:`system.profile.command <system.profile.command>` ``comment`` field
+       - :dbcommand:`currentOp` output, in the :data:`currentOp.command
+         <currentOp.command>` ``comment`` field
+
+       | For more information, see the :manual:`insert Command
+         Fields guide </reference/command/insert/#command-fields>` in the {+mdb-server+} manual.
+
+   * - ``hint``
+     - | Specifies the index to scan for documents that match the query filter.
+         For more information, see the
+         :manual:`Update Statement fields guide </reference/command/update/#std-label-update-statement-documents>`
```

Comment:
S: you can link directly to "hint" and make the link text a little more specific:
```suggestion
         :manual:`hint field </reference/command/update/#std-label-update-command-hint>`
```